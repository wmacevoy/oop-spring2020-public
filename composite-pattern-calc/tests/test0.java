class Greeter extends java.lang.Object
{
    Greeter() {};
    
    public static void main(java.lang.String args[])
    {
	java.lang.System.out.println("Hello World");
    }
    
}