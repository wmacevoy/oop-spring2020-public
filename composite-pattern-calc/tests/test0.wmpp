(2*3)
(y+3)
(303*z)
(l+335)
(0+p)
