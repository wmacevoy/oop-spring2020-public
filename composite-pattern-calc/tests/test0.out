6.0
3.0082645
2.4836063
335.00925
0.008928572
